JitsiMeetJS.js

```diff
@@ -283,9 +278,8 @@ export default {
                     = window.performance.now();
 
                 Statistics.analytics.sendEvent(
-                    addDeviceTypeToAnalyticsEvent(
-                        GET_USER_MEDIA_SUCCESS_, options),
-                    { value: options });
+                    createGetUserMediaEvent(
+                        'success', getAnalyticsAttributesFromOptions(options)));
```

Comment:
I don't know about you, it always makes me expend more effort than really necessary to mentally parse such lines than:
````javascript
createGetUserMediaEvent(
    'success',
    getAnalyticsAttributesFromOptions(options)));
````

Reply:
Agreed, fixed